fe/fe-core/src/main/java/com/starrocks/qe/SessionVariable.java

```diff
@@ -261,6 +262,8 @@ public class SessionVariable implements Serializable, Writable, Cloneable {
 
     public static final String PARSE_TOKENS_LIMIT = "parse_tokens_limit";
 
+    public static final String ENABLE_SORT_AGGREGATE = "enable_sort_aggregate";
```

Comment:
What's the connection between sort aggregate and streaming aggregate?
The noun is so misleading.

Reply:
I rename streaming_aggregator to sorted_streaming_aggregator